pandas/tests/groupby/test_nth.py

```diff
@@ -767,6 +767,29 @@ def test_groupby_nth_with_column_axis():
     tm.assert_frame_equal(result, expected)
 
 
+def test_groupby_nth_interval():
+    # 24205
```

Comment:
Can you add an indication this is a GitHub number, e.g. `# GH#24205`.

Reply:
Done.